internal/graphql/generated.go

```diff
@@ -0,0 +1,2413 @@
+
+// GetTotalCount returns QueryJupiterOneQueryV1QueryV1Response.TotalCount, and is useful for accessing the field via an interface.
+func (v *QueryJupiterOneQueryV1QueryV1Response) GetTotalCount() int { return v.TotalCount }
+
+// GetType returns QueryJupiterOneQueryV1QueryV1Response.Type, and is useful for accessing the field via an interface.
+func (v *QueryJupiterOneQueryV1QueryV1Response) GetType() string { return v.Type }
+
+// GetUrl returns QueryJupiterOneQueryV1QueryV1Response.Url, and is useful for accessing the field via an interface.
+func (v *QueryJupiterOneQueryV1QueryV1Response) GetUrl() string { return v.Url }
+
+// QueryJupiterOneResponse is returned by QueryJupiterOne on success.
+type QueryJupiterOneResponse struct {
+	QueryV1 QueryJupiterOneQueryV1QueryV1Response `json:"queryV1"`
+}
+
+// GetQueryV1 returns QueryJupiterOneResponse.QueryV1, and is useful for accessing the field via an interface.
+func (v *QueryJupiterOneResponse) GetQueryV1() QueryJupiterOneQueryV1QueryV1Response {
+	return v.QueryV1
+}
+
+type QueryV1Flags struct {
+	AllPages           bool `json:"allPages"`
+	ComputedProperties bool `json:"computedProperties"`
+	RowMetadata        bool `json:"rowMetadata"`
+	VariableResultSize bool `json:"variableResultSize"`
+}
+
+// GetAllPages returns QueryV1Flags.AllPages, and is useful for accessing the field via an interface.
+func (v *QueryV1Flags) GetAllPages() bool { return v.AllPages }
+
+// GetComputedProperties returns QueryV1Flags.ComputedProperties, and is useful for accessing the field via an interface.
+func (v *QueryV1Flags) GetComputedProperties() bool { return v.ComputedProperties }
+
+// GetRowMetadata returns QueryV1Flags.RowMetadata, and is useful for accessing the field via an interface.
+func (v *QueryV1Flags) GetRowMetadata() bool { return v.RowMetadata }
+
+// GetVariableResultSize returns QueryV1Flags.VariableResultSize, and is useful for accessing the field via an interface.
+func (v *QueryV1Flags) GetVariableResultSize() bool { return v.VariableResultSize }
+
+type SortOrder string
+
+const (
+	SortOrderAsc  SortOrder = "ASC"
+	SortOrderDesc SortOrder = "DESC"
+)
+
+type UpdateIntegrationInstanceInput struct {
+	Name                          string                                        `json:"name"`
+	SourceIntegrationInstanceId   string                                        `json:"sourceIntegrationInstanceId"`
+	PollingInterval               IntegrationPollingInterval                    `json:"pollingInterval"`
+	PollingIntervalCronExpression IntegrationPollingIntervalCronExpressionInput `json:"pollingIntervalCronExpression"`
+	Description                   string                                        `json:"description"`
+	Config                        map[string]interface{}                        `json:"config"`
+	OffsiteComplete               bool                                          `json:"offsiteComplete"`
+}
```

Comment:
It doesn't look like the generated code is going to handle optional fields very well. Not all integrations are supposed to have `PollingIntervalCronExpressions` and `SourceIntegrationInstanceId`. I'm not sure the API will smoothly handle the empty values Go will serialize.

Reply:
@zemberdotnet  Great catch! Thanks for looking over my shoulder. Take a peek now, I think it's closer to what we want.